ajax/libs/interactive-data-display/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "interactive-data-display",
+  "version": "1.5.0",
+  "filename": "idd.min.js",
+  "description": "A JavaScript visualization library for dynamic data",
+  "homepage": "https://github.com/predictionmachines/InteractiveDataDisplay/wiki",
+  "author": {
+    "name": "Information Technologies in Science Lab",
+    "url": "http://itis.cs.msu.ru/"
+  },
+  "keywords": [
+    "visualization",
+    "animation",
+    "canvas",
+    "chart",
+    "plot"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/predictionmachines/InteractiveDataDisplay.git"
+  },
+  "license": "Apache-2.0",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/predictionmachines/InteractiveDataDisplay.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*.!(d.ts)"
```

Comment:
@ElenaPochernina Sorry for my mistake. This config will also fetch `".d.ts"` files. Let's modify to `"**/*.!(*ts)"`. Thanks.